diff --git a/src/converter.ts b/src/converter.ts
index 3ccd197deab0514dd10e903d84e45bf589239370..39e1f0a03cf309556d167246a113f92bc7a3d185 100644
--- a/src/converter.ts
+++ b/src/converter.ts
@@ -2,7 +2,6 @@
 import bigDecimal = require('js-big-decimal')
 // @ts-check
 import numeral from 'numbro'
-import Web3 from 'web3'
 import converter from 'hex2dec'
 import { CHAIN_TYPE, CURRENCY_SYMBOL } from '@wallet/constants'
 
@@ -34,7 +33,7 @@ export const convertWeiToBalance = (strValue: string, iDecimal: string | number
   strValue = '' + strValue
 
   if(Number(strValue) === 0) return 0 ;
-  
+
   try {
     const decimalFormat = parseFloat(iDecimal.toString())
     const multiplyNum = new bigDecimal(Math.pow(10, decimalFormat))
@@ -75,7 +74,7 @@ export const formatFiatMoney = (price = '0') => {
 export const formatTokenBalance = (balance = '0', chain = '') => {
   const largeDecimalChains = [CHAIN_TYPE.ETH, CHAIN_TYPE.bitcoin]
 
-  if (largeDecimalChains.includes(chain)) 
+  if (largeDecimalChains.includes(chain))
     return formatNumberBro(balance, 8)
 
   return formatNumberBro(balance, 4)
@@ -138,7 +137,7 @@ export const calculateBigNumber = (arrValue: string[], method: 'add' | 'subtract
   try {
     if (arrValue) {
       const sumAmount = arrValue.reduce(
-        (preResult: bigDecimal, currentValue, currentIndex) => 
+        (preResult: bigDecimal, currentValue, currentIndex) =>
           currentIndex ? preResult[method](new bigDecimal(currentValue)) : new bigDecimal(currentValue), new bigDecimal(0)
       )
       return sumAmount.getValue()
@@ -212,7 +211,7 @@ export const roundingNumber = (number: number, rounding = 7) => {
 }
 
 export const convertCheckSUM = (address: string) => {
-  return Web3.utils.toChecksumAddress(address)
+  throw new Error('Function not elements')
 }
 
 export const convertDecimalToHex = (value: string) => {
@@ -273,7 +272,7 @@ export const formatBilion = (labelValue: string, mantissa = 2) => {
 /**
  * Compare between 2 number
  * @param from
- * @param to 
+ * @param to
  * @returns
  * - 1: from is higher
  * - 2: equal